src/ugeneui/html/version_news_en.html

```diff
@@ -18,14 +18,34 @@
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
  MA 02110-1301, USA.
 -->
-<h3>Changes in UGENE v. 39</h3>
+<h3>Changes in UGENE v. 40</h3>
+
+<h4>Alignment Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;">Multi-row selection support.</li>
+    <li style="margin-bottom: 10px;">The "Build Tree" dialog now has all PhyML algorithms listed.</li>
+    <li style="margin-bottom: 10px;">A new option to move selected sequences from one document to another.</li>
+    <li style="margin-bottom: 10px;">The grouping mode: sorting by the size of a group and showing group sizes in the name list.</li>
+</ul>
+
+<h4>Sanger Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;">A new visualization mode: show alternative mutations.</li>
+    <li style="margin-bottom: 10px;">Vertical scrolling bugs have been fixed.</li>
+</ul>
+
+<h4>Sequence Viewer</h4>
+<ul>
+    <li style="margin-bottom: 10px;">Improved speed, memory usage and precision of sequence graphs.</li>
+    <li style="margin-bottom: 10px;">Fixed stability problems of auto-annotation routines.</li>
+    <li style="margin-bottom: 10px;">Sequences with sizes above 2Gb now can be opened by UGENE again.</li>
+</ul>
+
+<h4>Other improvements</h4>
 <ul>
-    <li style="margin-bottom: 10px;"> Phylogenetic tree viewer now allows setting custom colors and fonts for subtree branches and labels.</li>
-    <li style="margin-bottom: 10px;"> The "Align to alignment" option got support for algorithm selection: ClustalO, MAFFT, Muscle, or UGENE.</li>
-    <li style="margin-bottom: 10px;"> Alignments with the RAW (undefined) alphabet can be converted to the DNA, RNA, or Amino alphabets.</li>
-    <li style="margin-bottom: 10px;"> Improved Unicode support for FASTA, MSF, and CLUSTALW formats.</li>
-    <li style="margin-bottom: 10px;"> Sequence Viewer got a new option to show codons or protein statistics for selected regions.</li>
-    <li style="margin-bottom: 10px;"> UGENE now speaks Turkish. Use menu Settings/Preferences to update the language.</li>
+    <li style="margin-bottom: 10px;">Improved CUDA build and error detection support.</li>
+    <li style="margin-bottom: 10px;">Unicode supported was added for Newick, Plain text, Diff, VCF, SNP file formats.</li>
+    <li style="margin-bottom: 10px;">Random Sequence Generator memory footprint and generation speed improvements.</li>
```

Comment:
`The "Random Sequence Generator" memory footprint and generation speed have been improved.`

Reply:
Done